src/components/molecules/new-select/components/control.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from "clsx"
+import React from "react"
+import { ControlProps, GroupBase } from "react-select"
+
+export const Control = <
+  T,
+  IsMulti extends boolean,
+  GroupType extends GroupBase<T>
+>({
+  children,
+  innerProps,
+  selectProps: { menuIsOpen },
+  innerRef,
+}: ControlProps<T, IsMulti, GroupType>) => {
+  return (
+    <div
+      {...innerProps}
+      ref={innerRef}
+      className={clsx(
+        "bg-grey-5 inter-base-regular w-full p-3 cursor-text border flex border-grey-20 focus-within:shadow-input focus-within:border-violet-60 rounded-rounded",
```

Comment:
Have you talked to Ludvig about the border radius? I can see he has changed it, in the revamp in January it was 4px, however, in the price list, customer group design it's 8px for this and all other inputs. If we change it we should probably create an issue to align the remaining input fields to ensure the border radius is consistent.

```suggestion
        "bg-grey-5 inter-base-regular w-full p-3 cursor-text border flex border-grey-20 focus-within:shadow-input focus-within:border-violet-60 rounded",
```